services/m3aggregator/config/aggregator.go

```diff
@@ -120,8 +120,11 @@ type AggregatorConfiguration struct {
 	// Client configuration for key value store.
 	KVClient kvClientConfiguration `yaml:"kvClient" validate:"nonzero"`
 
-	// KV namespace.
-	KVNamespace string `yaml:"kvNamespace" validate:"nonzero"`
+	// Election KV namespace.
+	ElectionKVNamespace string `yaml:"electionKVNamespace" validate:"nonzero"`
```

Comment:
Remove this and simply use KVConfig.Namespace below?

Reply:
Feels a little bit weird that way, although right now the namespace is the same for the placement and the election, but I feel they probably don't need to be tied together, also it's a bit weird that only namespace is used but not the environment. I could just use KVConfig.Namespace if you feel strongly.

Reply:
Agreed that the namespaces don't have to be the same, however I'd prefer to have a single namespace in the config until there is a need to have two separate namespaces for placement and election.

Reply:
Sounds good